### DISCUSSION TITLE
Error when creating executable file using tkinter and a flask.

### DISCUSSION THREAD

**Discussion by JjayFabor** [pyinstaller/pyinstaller#8062] [Help]
Hello, I created a project of flask web application. I cannot deploy or host the flask web application, so I created a tkinter application that have login system then after logging in, it will open the web browser for flask web application. However, when I use pyinstaller to make it as an executable file, the flask web application won't reload or the site can't be reached. Is there something wrong with me using the pyinstaller?

**Comment by rokm** [comment]
That's nowhere near enough information for us to help you. We need either the error message (which, if non-fatal, is likely written to console, so make sure you do *not* use `--windowed` / `--noconsole` option if building on Windows), or self-contained example to reproduce the issue locally.
